lib/components/narrative/line-itin/connected-itinerary-body.js

```diff
@@ -134,6 +134,10 @@ class ConnectedItineraryBody extends Component {
             setMainPanelContent(MainPanelContent.TRIP_VIEWER)
           }}
           showAgencyInfo
+          showApproximatePrefixNonTransitLegs={
+            config.itinerary &&
+            config.itinerary.showApproximatePrefixNonTransitLegs
+          }
```

Comment:
Does this work with the types you have set up?
```suggestion
          showApproximatePrefixNonTransitLegs={
            config.itinerary?.showApproximatePrefixNonTransitLegs
          }
```

Reply:
I added a new type to config-types.js